internal/docs/readme.go

```diff
@@ -17,27 +17,24 @@ import (
 	"github.com/elastic/elastic-package/internal/packages"
 )
 
-// ReadmeFile for the Elastic package
-const ReadmeFile = "README.md"
-
 // IsReadmeUpToDate function checks if the README file is up-to-date.
-func IsReadmeUpToDate() (bool, error) {
-	logger.Debugf("Check if %s is up-to-date", ReadmeFile)
+func IsReadmeUpToDate(readmeFileName string) (bool, error) {
```

Comment:
Then, methods which won't be used externally can be unexposed.

Reply:
Yep! Now with the change, `IsReadmeUpToDate()` stays the same without the file name as input parameter.